manifest.yml

```diff
@@ -0,0 +1,13 @@
+declared-services:
```

Comment:
The file should start with `---`: or else the declared services won't be recognized:

```
---
declared-services:
  sljsp-cloudantNoSQLDB:
    label: cloudantNoSQLDB
    plan: Lite
applications:
...
```